multipathd/multipathd-shutdown-if-idle.timer

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Device-Mapper Multipath Conditional Shutdown Trigger
+After=multi-user.target
```

Comment:
To be certain, we should als add "After=multipathd.service" here IMO

Reply:
ACK.